### ISSUE TITLE
Пишу на русском. Попытался создать на Русском языке алиас "New-Item". Криво... надо поправлять.

### ISSUE THREAD

**Issue by MironAtHome** [MicrosoftDocs/PowerShell-Docs.ru-ru#1]
Hey Jim,

alias in Russian for command "New-Item" doesn't really work.
Consider this sequence of commands:

New-Alias Новая-Вещь New-Item
New-Alias Папка Directory
New-Alias -Папка -Directory

I know... I am playing an dopey. The way it's "done" is 

New-Alias "Новая-Папка"  "New-Item -Type Directory"

and voila, all is well. But heck, why not a write a whole new language, if I have to play a dopey to the ways of how it's "done". I would prefer Powershell does its part fairly, so that I can write in Russian, as a first row citizen language, without switching keyboard back and forth. In the end I am "paying" for Windows money, that I earn, and I would think that paying for something means getting work done for my money to my liking. Familiar with the concept?
Take for example command line. 15 years ago command line would write in Russian, if you set all the knobs to the "t" ( the "t" was rather considerable ), until NT4.0 service pack 5, edition 2, an attempt to save text in notepad was futile, unless you install fully localized OS, which came with about 100 pages worth of hardware compatibility requirements. Today I install Windows 10, add Russian language in regional settings, set Russian language system default, and I have my mother's tongue environment on my computer. My directory tree, except drive C ( think about it, if it keeps on going that way, eventually Lynux wins by default ) is in Russian language. I intentionally skipped system subfolders. Gosh... it took from me about 5 minutes sitting with my eyes pointing towards the lighting fixture in the ceiling, to consider what am I missing. Why do I do it? Ah... because there is an engineer who will look at this request and write back "hey dopey, you got a whole C:\Windows directory _IN_ENGLISH_ on your machine. So... tough luck." If only this engineer could explain, why I as a user would ever step into that directory... But I do need to get under drive "C" rather frequently. And every time I do that, I have to perform switch of the language context. At least today.
In other words, this is a call for unabashed rethinking and creative work. Rather than analytical as to "why that request is wrong". And if you tell me "I ain't got a budget for that", well, buddy, look around!

Most cordially all the best,

Miron.

**Comment by MironAtHome** [comment]
Leave a comment

Gosh... I thought this gets it off my chest and I am free to go..

Look at this, Jim:

C:\зодчий\исток\гит> git clone https://github.com/PowerShell/powerShell-Docs.ru-ru.git
Cloning into 'powerShell-Docs.ru-ru'...
fatal: unable to access 'https://github.com/PowerShell/powerShell-Docs.ru-ru.git/': error setting certificate verify locations:
  CAfile: C:\Users\raisa\AppData\Local\GitHub\PortableGit_25d850739bc178b2eb13c3e2a9faafea2f9143c0\mingw32/usr/ssl/certs/ca-bundle.crt
  CApath: none

I can't even have my directory tree in Russian language, at the fear of not being able to use Git repositories with source repository support... You want to tell me "use zip download". Well... I got just a few... no... a few more than that answers. None of them can be published.

**Comment by vors** [comment]
Hi Miron,

I think this is not a documentation problem, but the powershell itself problem. So it's better to move the issue into https://github.com/PowerShell/PowerShell
PowerShell aliases are not that flexible compare to bash for example and don't allow to alias specialised version for parameters.
I created a project to workaround this here https://github.com/vors/ParamAlias
It's more like a proof of concept then a working solution and I definitely didn't test unicode support there, but it seems relevant here.
